src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -309,15 +325,20 @@ public override async Task<BaseConfiguration> GetBaseConfigurationAsync(Cancella
         /// </summary>
```

Comment:
The doc above needs to be updated.

Also, can we add a check `RefreshInterval == TimeSpan.MaxValue` to do an early return?